packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -67,7 +68,6 @@ const CFDsListing = observer(() => {
     const { setAccountType, toggleCTraderTransferModal } = cfd;
     const {
         account_status,
-        ctrader_accounts_list,
```

Comment:
Removing as it wasn't used anywhere 
<img width="800" alt="Screenshot 2024-05-07 at 3 46 29 PM" src="https://github.com/binary-com/deriv-app/assets/64970259/29118caf-f8b4-4735-a605-3a3275151a80">